include/ofi_shm.h

```diff
@@ -330,15 +331,15 @@ int	smr_map_create(const struct fi_provider *prov, int peer_count,
 		       struct smr_map **map);
 int	smr_map_to_region(const struct fi_provider *prov,
 			  struct smr_peer *peer_buf);
-void	smr_map_to_endpoint(struct smr_region *region, int index);
-void	smr_unmap_from_endpoint(struct smr_region *region, int index);
+void	smr_map_to_endpoint(struct smr_region *region, int64_t index);
+void	smr_unmap_from_endpoint(struct smr_region *region, int64_t index);
```

Comment:
Without looking into these functions, would renaming index to id be more consistent?